converter/src/main/java/gov/cms/qpp/conversion/util/FormatHelper.java

```diff
@@ -0,0 +1,37 @@
+package gov.cms.qpp.conversion.util;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+public class FormatHelper {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)

Reply:
Sonar is slow to update I think. This has already been added.